### DISCUSSION TITLE
Use with WP-SCSS plugin

### DISCUSSION THREAD

**Discussion by bnikkhah** [holger1411/understrap#1748] [Q&A]
Hello,

Is it possible to edit UnderStrap (and UnderStrap child) SASS directly in WordPress using WP-SCSS plugin? (https://wordpress.org/plugins/wp-scss/)

Thanks!

**Comment by iwasthesword** [comment]
Maybe it could be done if you set it up just right, but I've been using [bootscore/bootscore](https://github.com/bootscore/bootscore) which has this same functionality built-in

**Comment by IanDelMar** [comment]
While I think it is technically possible, you will lose the editor color palette which is generated during the build process. Bootstrap 5 comes with a couple of [root variables](https://github.com/twbs/bootstrap/blob/547aa099ead7e4fa1ee99efedf44fbf346bb8394/dist/css/bootstrap.css#L8-L75). Depending on your needs maybe it's sufficient to adjust those via CSS?
